rotkehlchen/externalapis/coingecko.py

```diff
@@ -408,11 +409,13 @@ def _query(
                 raise RemoteError(f'Coingecko API request failed due to {str(e)}') from e
 
             if response.status_code == 429:
-                # Coingecko allows only 100 calls per minute. If you get 429 it means you
+                # Coingecko allows only 50 calls per minute. If you get 429 it means you
                 # exceeded this and are throttled until the next minute window
-                # backoff and retry 4 times =  2.5 + 3.33 + 5 + 10 = at most 20.8 secs
+                # backoff and retry COINGECKO_QUERY_RETRY_TIMES times
+                # that is 1.67 + 2.5 + 5 = at most 9.17 secs
```

Comment:
```suggestion
                # that is 1.67 + 2.5 + 5 = at least 9.17 secs
```

think this was wrong before too